### ISSUE TITLE
Website fails when Image Standardization is disabled

### ISSUE THREAD

**Issue by JohnEEnglish** [waynebhayes/SpArcFiRe#14]

**Comment by AlanBarkleyYeung** [comment]
I think that I resolved the issue. The website should now make zip files even if SpArcFiRe fails to make all the images during execution.

**Comment by waynebhayes** [comment]
OK.... so do we need to update the code in the github repo?
--
Wayne Hayes, Ph.D.
Associate Professor of Computer Science, University of California, Irvine
Director, UCI-SDSU Joint Ph.D. Program in Computational Science (UCI side)
*The evidence is now incontrovertible that Russia interfered with the 2016
US Presidential election by manipulating public opinion with disinformation
via thousands of fake social media accounts. Why? Follow the money: Trump
has been laundering it for Russian billionaires, including Putin, for 20
years. Details and fact checking
at https://secure.avaaz.org/campaign/en/shocking_truth_about_trump/?bebhylb
<https://secure.avaaz.org/campaign/en/shocking_truth_about_trump/?bebhylb>.*
The opinions expressed in this email by Professor Hayes are his and his
alone, and are not meant to represent those of the University of California.

**Comment by AlanBarkleyYeung** [comment]
I don't believe so. All the changes that I made were in the php files, not SpArcFiRe itself.

**Comment by wschallo** [comment]
Pedro & I stumbled upon this:
**Image dimensions must be an even number**

if not set parameter: useImageStanardization 1

see SpArcFiRe/matlab/mergeOriFields.m line 20

**Comment by waynebhayes** [comment]
Interesting. Is there any particular reason why the dimensions must be an even number? Darren must have had a good reason so it's probably not worth pursuing, but this error message should be passed on to the user of the website (or there should be a note somewhere as the user uploads). Alternately we could quietly lop off one row or column of pixels arbitrarily but that's a bit under-handed.
